package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
+    "react-router-dom": "^6.27.0",
```

Comment:
as a rule of thumb, I'd try to make sure everyone has the same package.json file and avoid changes to it - this will just ensure that everything stays consistent!